web/src/components/network/NetworkPage.jsx

```diff
@@ -58,135 +43,117 @@ const NoWiredConnections = () => {
+      if ([NetworkEventTypes.DEVICE_ADDED, NetworkEventTypes.DEVICE_UPDATED, NetworkEventTypes.DEVICE_REMOVED].includes(type)) {
+        setUpdateState(true);
       }
-      client.connections().then(setConnections);
     });
-  }, [client, devices]);
-
-  useEffect(() => {
-    if (devices !== undefined) return;
+  });
 
-    client.devices().then(setDevices);
-  }, [client, devices]);
-
-  const selectConnection = ({ id }) => {
-    client.getConnection(id).then(setSelectedConnection);
-  };
+  const connectionDevice = ({ id }) => devices?.find(({ connection }) => id === connection);
+  const connectionAddresses = (connection) => {
+    const device = connectionDevice(connection);
+    const addresses = device ? device.addresses : connection.addresses;
 
-  const forgetConnection = async ({ id }) => {
-    await client.deleteConnection(id);
-    setConnections(undefined);
-  };
-
-  const updateConnections = async () => {
-    setConnections(undefined);
-    setDevices(undefined);
+    return addresses?.map(formatIp).join(", ");
   };
 
   const ready = (connections !== undefined) && (devices !== undefined);
 
   const WifiConnections = () => {
-    const wifiConnections = connections.filter(c => c.wireless);
+    const { wireless_enabled: wifiAvailable } = settings;
 
-    if (wifiConnections.length === 0) {
+    if (!wifiAvailable) {
       return (
-        <NoWifiConnections wifiScanSupported={settings.wireless_enabled} />
+        <CardField>
+          <CardField.Content>
+            <EmptyState title={_("No WiFi support")} icon="wifi_off" color="warning-color-200">
+              {_("The system does not support WiFi connections, probably because of missing or disabled hardware.")}
+            </EmptyState>
+          </CardField.Content>
+        </CardField>
       );
     }
 
+    const wifiConnections = connections.filter(c => c.wireless);
+    const activeWifiDevice = devices.find(d => d.type === "wireless" && d.state === "activated");
+    const activeConnection = wifiConnections.find(c => c.id === activeWifiDevice?.connection);
+
     return (
-      <ConnectionsTable connections={wifiConnections} devices={devices} onEdit={selectConnection} onForget={forgetConnection} />
+      <CardField
+        label={_("Wi-Fi")}
+        actions={
+          <ButtonLink isPrimary={!activeConnection} to="wifis">
+            {activeConnection ? _("Change") : _("Connect")}
+          </ButtonLink>
+        }
+      >
+        <CardField.Content>
+          {activeConnection
+            ? (
+              <EmptyState title={sprintf(_("Conected to %s"), activeConnection.id)} icon="wifi" color="success-color-100">
+                {connectionAddresses(activeConnection)}
+              </EmptyState>
+            )
+            : (
+              <EmptyState title={_("No connected yet")} icon="wifi_off" color="color-300">
+                {_("The system has not been configured for connecting to a WiFi network yet.")}
```

Comment:
Please, use `Wi-Fi` (refer to [SUSE Documentation Style Guide](https://documentation.suse.com/style/current/single-html/docu_styleguide/index.html)).